includes/Assets.php

```diff
@@ -319,6 +319,10 @@ public function get_scripts() {
                 'deps'      => [ 'jquery' ],
                 'in_footer' => false,
             ],
+            'dokan-recaptcha' => [
```

Comment:
`dokan-recaptcha` key name should be change because if we introduce another recapcha provider then problem will be there of naming. You can use `dokan-google-recaptcha`.